src/js/common/schemaform/SaveFormLink.jsx

```diff
@@ -1,8 +1,20 @@
 import React from 'react';
+import Scroll from 'react-scroll';
 import PropTypes from 'prop-types';
 
 import LoginModal from '../components/LoginModal';
-import { SAVE_STATUSES } from './save-load-actions';
+import { SAVE_STATUSES, saveErrors } from './save-load-actions';
+import { focusElement } from '../utils/helpers';
+
+const Element = Scroll.Element;
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
```

Comment:
Yeah, it's tempting, but we normally need to put an `Element` on the page, too, and you're not saving a whole lot if you still have to important react-scroll wherever you're using it.